DataManagementSystem/Client/DataManager.py

```diff
@@ -184,7 +184,7 @@ def __cleanDirectory(self, folder):
                 "%s %s" % (lfn, reason))
     res = returnSingleResult(self.removeFile(['%s/dirac_directory' % folder]))
     if not res['OK']:
-      if not "No such file" in res['Message']:
+      if "No such file" not in res['Message']:
```

Comment:
cmpError could be used here

Reply:
which error code should be used?

Reply:
`ENOENT` is the standard for `No such file or directory`